wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsViewController.swift

```diff
@@ -340,3 +342,18 @@ extension GroupDetailsViewController: GroupDetailsSectionControllerDelegate, Gro
     }
 
 }
+
+extension ZMConversation {
+
+    func refetchParticipantsIfNeeded() {
+        guard let session = ZMUserSession.shared() else {
```

Comment:
I don't think the `ZMConversation` should really know about the `ZMUserSession`, I would suggest to do something like: 

```swift
session.perform {
    conversation.refetchParticipantsIfNeeded()
}
```

Reply:
done